experimental/PropertyDDS/packages/property-shared-tree-interop/src/test/tableExampleSchemaConverter.spec.ts

```diff
@@ -148,28 +146,27 @@ function checkMissingRefs(schemaData) {
 }
 
 function checkInheritanceTranslation(schemaData) {
-	const schemaMap = schemaData.treeSchema;
-	const row = schemaMap.get("array<Test:Row-1.0.0>");
-	expect(row).not.toBeUndefined;
-	expect(row?.localFields).not.toBeUndefined;
-	const field = row?.localFields.get("");
-	expect(field).not.toBeUndefined;
-	expect(field?.types).not.toBeUndefined;
-	const types = field?.types;
-	expect(types?.has("Test:Row-1.0.0")).toBeTruthy;
-	expect(types?.has("Test:ExtendedRow-1.0.0")).toBeTruthy;
-	expect(types?.has("Test:OtherExtendedRow-1.0.0")).toBeTruthy;
+	const row = lookupTreeSchema(schemaData, brand("array<Test:Row-1.0.0>"));
+	assert(row !== undefined);
+	assert(row.localFields !== undefined);
+	const field = row.localFields.get(EmptyKey);
+	assert(field !== undefined);
+	assert(field.types !== undefined);
+	const types = field.types;
+	expect(types.has(brand("Test:Row-1.0.0"))).toBeTruthy();
+	expect(types.has(brand("Test:ExtendedRow-1.0.0"))).toBeTruthy();
+	expect(types.has(brand("Test:OtherExtendedRow-1.0.0"))).toBeTruthy();
```

Comment:
Please, be aware that I've only changed this particular test to show what I mean in my review. If you agree with this approach regarding `assert`, I would suggest to rework all other tests as well.

Reply:
> I have no problem with this.

So, should we rework the tests here or in your PR?

Reply:
Or we can even merge this PR and perform the rework in the next PR. I can focus on this at the end of next week at first and would like to have this in ASAP. Or do you think that the tests are invalid right now?